src/Icicle/Sea/FromAvalanche/Prim.hs

```diff
@@ -165,6 +165,8 @@ seaOfPrimArray p
       -> PDAlloc (prefixOfValType (ArrayT t) <> "put_immutable") Nothing
      PrimArraySwap t
       -> PDFun   (prefixOfValType (ArrayT t) <> "swap") Nothing
+     PrimArrayDel t
+      -> PDFun   (prefixOfValType (ArrayT t) <> "delete") Nothing
```

Comment:
yeah, though I wonder if instead of `put_immutable` we could just have `put_immutable = put_mutable . copy`. then we'd only need one version of each mutable array operation. would that complicate mutation recovery though?